src/styles.css

```diff
@@ -132,3 +143,72 @@ h3 {
   }
 }
 
+#clickedRecipe {
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  right: 0;
+  width: 98%;
+  border-radius: 50px;
+  background-color: darkgray;
```

Comment:
Only if you feel like, you could try playing around with the opacity (between 0 and 1) of these backgrounds. Just so that you've explored more looks.